tests/test_parts.py

```diff
@@ -33,6 +33,7 @@ def test_all():
     parts = Parts(channel)
 
     helper_test_update_parts_list(parts)
+    time.sleep(1)
```

Comment:
The test `helper_test_update_parts_from_AVL()` fails sometimes without a delay. This indicates that the update parts list API is returning the response before done processing.